### ISSUE TITLE
Automatically set the body type

### ISSUE THREAD

**Issue by mittsh** [luckymarmot/Paw-HARImporter#1]
Paw lets users set different body types (plain text, form url encoded, JSON, multipart). It would be nice to detect which body type is most appropriate depending on the `Content-Type` header.

Here is the API in details: https://luckymarmot.com/paw/doc/Extensions/Reference/Request#urlencodedbody

Though, I'm thinking that maybe should we factor this in a new common API like:

``` javascript
request.setAppropriateBody(body_string, content_type)
```

It would work with:

``` javascript
request.setAppropriateBody("{\"value\":42}", "application/json")
```

... @jeroenvisser101 what do you think? is it worth a new JS API, or just do the parsing in this code?

**Comment by mittsh** [comment]
> It would be easier to have this in one place I guess, as it's central and accessable for all plugins. All plugins would behave in the same way, too. But whether to add it as a NPM package and then check, or add a JS API, I'm not sure...

@jeroenvisser101 – https://github.com/jeroenvisser101/Paw-HARImporter/issues/1

**Comment by JonathanMontane** [comment]
fixed in PR #4
